modules/nf-core/vsearch/fastqfilter/tests/main.nf.test

```diff
@@ -0,0 +1,59 @@
+nextflow_process {
+
+    name "Test Process VSEARCH_FASTQFILTER"
+    script "../main.nf"
+    process "VSEARCH_FASTQFILTER"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "vsearch"
+    tag "vsearch/fastqfilter"
+
+    test("sarscov2 - fastq") {
+
+        when {
+            process {
+                """
+                input[0] = [
+                    [id:'test'], // meta map
+                    file(params.modules_testdata_base_path + 'genomics/sarscov2/illumina/fastq/test_1.fastq.gz', checkIfExists: true),
+                ]
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(process.out).match() }
```

Comment:
Haven't looked what the log contains, but something like this would hopefully snapshot the first line. See what is not varying.
```suggestion
                { assert snapshot(
                  process.out.fasta,
                  file(process.out.log[0][1]).readLines[0],
                  process.out.versions,
                ).match() }
```